freqtrade/exchange/exchange.py

```diff
@@ -818,6 +827,7 @@ def create_order(self, pair: str, ordertype: str, side: str, amount: float,
             rate_for_order = self.price_to_precision(pair, rate) if needs_price else None
 
             self._lev_prep(pair, leverage)
+            amount = self._amount_to_contract_size(pair, amount)
```

Comment:
I think this should be combined into `amount_to_precision` (roughly 5 lines above) - which does exactly the same thing.